src/main/resources/db/migration/V4_5__Create_ingestion_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS ingestion
```

Comment:
1. rename DB table name to `adaptors_details`.
2. add `created_at` and `modified_at` columns
3. insert `created_at` and `modified_at` columns through trigger on insert/update